crates/bitwarden/src/vault/cipher/cipher.rs

```diff
@@ -43,6 +43,8 @@ pub struct Cipher {
     pub folder_id: Option<Uuid>,
     pub collection_ids: Vec<Uuid>,
 
+    pub key: Option<EncString>,
```

Comment:
Can we add a comment here what the key is.

```suggestion
    /// More recent ciphers uses individual encryption keys to encrypt the other fields of the Cipher.
    pub key: Option<EncString>,
```